mater/lib/src/v2/mod.rs

```diff
@@ -82,7 +87,7 @@ impl Header {
     /// The [`Header`] size in bytes (includes the pragma).
     ///
     /// As defined in the [specification](https://ipld.io/specs/transport/car/carv2/#header).
-    pub const SIZE: usize = PRAGMA.len() + 40;
+    pub const SIZE: u64 = PRAGMA_SIZE + 40;
```

Comment:
Why conversion to u64? I don't think size can be < 0.

Reply:
u64 is unsigned. I've added casting here so that it's not needed at the call site.